pages/_app.js

```diff
@@ -27,21 +27,29 @@ export default function App({ Component, pageProps }) {
   const [dayTrending, setDayTrending] = useState(true);
   const [search, setSearch] = useState("");
   const [theme, setTheme] = useState("light");
+  const [resultsPage, setResultsPage] = useState(1);
 
-  const movies = useLocalStorageFetch(
-    `https://api.themoviedb.org/3/search/movie?api_key=${process.env.NEXT_PUBLIC_API_KEY}&language=eng-US&query=${search}`,
+  const moviesData = useLocalStorageFetch(
+    `https://api.themoviedb.org/3/search/movie?api_key=${process.env.NEXT_PUBLIC_API_KEY}&language=eng-US&query=${search}&page=${resultsPage}`,
```

Comment:
You shouldn't use your API_KEY in client side code. It's visible to everyone!
Please use an environment variable not starting with the `NEXT_PUBLIC` prefix and build your own API that uses this key. Route your MovieDB API requests through your own API then.